.changes/next-release/Feature-54185f1e-ed33-4b8e-b0e9-0a5a1a3dccb1.json

```diff
@@ -0,0 +1,4 @@
+{
+    "type": "Feature",
+    "description": "Added openDocumentLocally and publishDocument commands, removed support for command type documents"
```

Comment:
The ssm document feature branch as a whole only needs one changelog entry.

Reply:
Will delete previous changelog entry

Reply:
In that case we should remove `removed support for command type documents`. ?

Reply:
I deleted the previous one, the changelog that did not include "removed support for command type documents"

Reply:
Consider removing all ssm doc change logs, and produce an appropriate singular entry when the feature branch nears completion.